### PR TITLE
Added Image2icon

### PR DESCRIPTION
**Pull request by highvoltag3**
Nice little utility to create icons for Mac. Great for quickly
exporting retina versions for development, but could also produce icons to customize your Mac.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -161,6 +161,7 @@
 - [Affinity Designer](https://affinity.serif.com/en-us/designer/) - Vector image design tool, possible Adobe Illustrator alternative.
 - [Affinity Photo](https://affinity.serif.com/en-us/photo/) - Raster image design tool, possible Adobe Photoshop alternative.
 - [Blender](https://www.blender.org/) - Fully-featured extensible cross-platform 3D content suite. [![Open-Source Software][OSS Icon]](https://developer.blender.org/)
+- [Image2iCon](http://www.img2icnsapp.com) - Mac tool that allow you to create and personalize icons from your pictures.
```
Comment (iCHAIT) [inline README.md]: No need to specify `mac` in the description.

**Comment by iCHAIT** [comment]
This is pretty cool :+1: 

Will merge once you fix the comment I left.

**Comment by highvoltag3** [comment]
@iCHAIT all set.

**Comment by iCHAIT** [comment]
:+1:

**Pull request merged**